Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -608,3 +607,24 @@ void HexEditorWidget::drop_event(GUI::DropEvent& event)
         open_file(response.value().filename(), response.value().release_stream());
     }
 }
+
+DeprecatedString HexEditorWidget::make_display_string(DeprecatedString string)
```

Comment:
No need to take a copy here, either take a const ref or even a `StringView`.

Reply:
done 👍

What would taking a `StringView` look like here? You mean changing the declaration to

```c++
StringView HexEditorWidget::make_display_string(StringView const& string)
 ```

Would that require casting the args to `StringView`? Like

```c++
make_display_string(StringView { ascii_string });
```

What would be the advantages of that? They are not clear to me

Reply:
What would taking a `StringView` look like here? You mean changing the declaration to

```c++
StringView HexEditorWidget::make_display_string(StringView const& string)
 ```

Would that require casting the args to `StringView`? Like

```c++
make_display_string(StringView { ascii_string });
```

What would be the advantages of that? They are not clear to me

Reply:
done 👍